relis_deployment/README.md

```diff
@@ -1,64 +1,149 @@
+# ReLiS
```

Comment:
This is the readme of the project, but it should be the readme of the Docker deployment